fairing/builders/base_builder.py

```diff
@@ -51,7 +51,10 @@ def generate_pod_spec(self):
                 env=[client.V1EnvVar(
                     name='FAIRING_RUNTIME',
                     value='1',
-                )]
+                )],
+                # Set the directory where the python files are built.
+                # TODO(jlewi): Would it be better to set PYTHONPATH?
+                working_dir="/app",
```

Comment:
this should be `self.preprocessor.path_prefix`

Reply:
Done